example/bricks-example-custom-analyzers/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.github.ingogriebsch.bricks</groupId>
+        <artifactId>bricks</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>bricks-example-custom-analyzers</artifactId>
+
+    <name>Example for company specific analyzers</name>
+    <description>If you have company-specific conventions, it might make sense to build analyzers to gather extra info from them. This is an example project that shows how easy it is to extend the Bricks analysis with own custom analyzers.</description>
+
+    <properties>
+        <java.version>1.8</java.version>
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.github.ingogriebsch.bricks</groupId>
+            <artifactId>bricks-maven-plugin</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
+        </dependency>
```

Comment:
Even if an example, this looks a little bit like that we need to have the plugin as a dependency.

Reply:
not at all, as it extends the plugin. Maybe i don't get your concern?